src/pipedrive-provider.ts

```diff
@@ -0,0 +1,51 @@
+const pipedrive = require("pipedrive"); // or however you import pipedrive
+
+const defaultClient = new pipedrive.ApiClient();
+
+// Configure authorization by setting API key
+// PIPEDRIVE_API_KEY is an environment variable that holds the real API key
+defaultClient.authentications.api_key.apiKey =

```

Comment:
do not commit api keys

Reply:
The api key was moved during our last session

Reply:
We also talked about this in our last session and has been addressed. I won't be doing that again.